src/env.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const {env, MaybeType, EitherType} = require('sanctuary');
+const $ = require('sanctuary-def');
+const {test, TypeVariable, NullaryType, BinaryType, UnaryTypeVariable} = $;
+const F = require('fluture');
+
+const F_DOCS = 'https://github.com/Avaq/Fluture';
+const COMPARE_DOCS = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/'
+  + 'Reference/Global_Objects/Array/sort#Description';
+
+const last = xs => xs[0];
+const is = (T, x) => test(exports.env, T, x);
```

Comment:
This is the only reason I'm waiting for https://github.com/sanctuary-js/sanctuary/pull/354, and I might as well pass `[]` instead of `env`, but I prefer to "do it right" from the start.